xdsl/dialects/test.py

```diff
@@ -153,6 +153,92 @@ def __init__(
         )
 
 
+@irdl_op_definition
```

Comment:
Big fan of adding new operations to the test dialect

Reply:
That - and the algorithms - is all copy-pasta-porta from MLIR implementations :yum: